src/dashboard/src/dispatch.c

```diff
@@ -108,6 +108,10 @@ void dispatch_init() {
     carstats.vcu_controls_received = false;
     carstats.vcu_lc_controls_received = false;
 
+    carstats.rpm_setpoint.rpm_setpoint = 0;
+    carstats.ki.ki_times_1000 = 1000;
```

Comment:
I'd recommend you just zero this when init'ing

Reply:
Or loading in the default vals (we might want to start putting kp, ki as defines in a header for speed controller